### PR TITLE
Move common parts of commands to own class

### PR DESCRIPTION
**Pull request by beatrycze-volk**
Small quality improvements to Command classes

### CODE REVIEW

File: Classes/Command/IndexCommand.php [hunk 1]
```diff
@@ -88,10 +87,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $io = new SymfonyStyle($input, $output);
         $io->title($this->getDescription());
 
-        $startingPoint = 0;
-        if (MathUtility::canBeInterpretedAsInteger($input->getOption('pid'))) {
-            $startingPoint = MathUtility::forceIntegerInRange((int) $input->getOption('pid'), 0);
-        }
+        $startingPoint = getStartingPoint($input->getOption('pid'));
```
Comment (albig) [inline Classes/Command/IndexCommand.php]:
```suggestion
        $startingPoint = $this->getStartingPoint($input->getOption('pid'));
```

File: Classes/Command/ReindexCommand.php [hunk 2]
```diff
@@ -94,10 +94,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $io = new SymfonyStyle($input, $output);
         $io->title($this->getDescription());
 
-        $startingPoint = 0;
-        if (MathUtility::canBeInterpretedAsInteger($input->getOption('pid'))) {
-            $startingPoint = MathUtility::forceIntegerInRange((int) $input->getOption('pid'), 0);
-        }
+        $startingPoint = getStartingPoint($input->getOption('pid'));
```
Comment (albig) [inline Classes/Command/ReindexCommand.php]:
```suggestion
        $startingPoint = $this->getStartingPoint($input->getOption('pid'));
```

**Review by albig — CHANGES_REQUESTED** [review]
I can't get this working. Even with my two suggested changes, I run into this:

```
Uncaught TYPO3 Exception Return value of Kitodo\Dlf\Command\BaseCommand::getSolrCoreUid() must be of the type int, null returned
thrown in file /var/www/html/public/typo3conf/ext/dlf/Classes/Command/BaseCommand.php
in line 61
```

**Comment by beatrycze-volk** [comment]
> I can't get this working. Even with my two suggested changes, I run into this:
> 
> ```
> Uncaught TYPO3 Exception Return value of Kitodo\Dlf\Command\BaseCommand::getSolrCoreUid() must be of the type int, null returned
> thrown in file /var/www/html/public/typo3conf/ext/dlf/Classes/Command/BaseCommand.php
> in line 61
> ```

I have found a problem. I was missing _Id_ in the parameter's name for method `getSolrCoreUid`.

I have also inserted your suggestion in my fix commit.

**Review by albig — APPROVED** [review]
Seems to work now. Thank you.

**Pull request merged**